### PR TITLE
Add applause effect to results screen

### LINKED ISSUE #10571: Add "applause" support
**Describe the new feature:** 
So far there hasn't been a single request for this. But this is an important part of the skin.

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on https://github.com/ppy/osu/pull/10614
- [x] Depends on https://github.com/ppy/osu-resources/pull/116
- [x] Depends on https://github.com/ppy/osu/pull/10619
- Closes https://github.com/ppy/osu/issues/10571

Note that this doesn't add beatmap skin support. Will look at that later since it's not used in many scenarios and requires a considerable amount of implementation.

### CODE REVIEW

File: osu.Game/Audio/HitSampleInfo.cs [hunk 1]
```diff
@@ -50,9 +50,9 @@ public virtual IEnumerable<string> LookupNames
             get
             {
                 if (!string.IsNullOrEmpty(Suffix))
-                    yield return $"{Bank}-{Name}{Suffix}";
+                    yield return $"Gameplay/{Bank}-{Name}{Suffix}";
 
-                yield return $"{Bank}-{Name}";
+                yield return $"Gameplay/{Bank}-{Name}";
```
Comment (bdach) [inline osu.Game/Audio/HitSampleInfo.cs:55]: I believe moving these prefixes locally will break every single legacy skin (including beatmap skins) with custom gameplay hitsounds, unfortunately.

**Review by bdach — CHANGES_REQUESTED** [review]

File: osu.Game/Audio/HitSampleInfo.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Audio/HitSampleInfo.cs:55]: legacy skins already do local manipulation to remove all prefixes

Comment (peppy) [reply osu.Game/Audio/HitSampleInfo.cs:55]: Should be resolved (via https://github.com/ppy/osu/pull/10619).

**Review by bdach — DISMISSED** [review]
looks good to me :+1:

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**